common/src/main/java/com/viaversion/viabackwards/protocol/v1_21_6to1_21_5/data/Dialog.java

```diff
@@ -159,16 +159,23 @@ private void fillDialogList(final RegistryAndTags registryAndTags, final ServerL
                 dialogsTag.add(dialogTag);
             }
 
+            ListTag<StringTag> registryDialogsTag = tag.getListTag("dialogs", StringTag.class);
```

Comment:
For the command in line 153:
// Hold them as either a list of inlined / singleton inlined, a list of registry entries / singleton entry or a tag.